generators/client/templates/angular/src/main/webapp/app/shared/_shared-libs.module.ts

```diff
@@ -9,12 +9,13 @@ import { InfiniteScrollModule } from 'angular2-infinite-scroll';
 @NgModule({
     imports: [
         NgbModule.forRoot(),
-        NgJhipsterModule.forRoot({
+        // TODO uncomment the code below and make it works with AOT
+        NgJhipsterModule/*.forRoot({
```

Comment:
Yep I tried with a small app, working well with AOT but when I include ng-jhipster it doesn't work anymore. Maybe it doesn't come from the library itself but from the way we call it.